### PR TITLE
Bump nginx ingress version to 0.9.0-beta.1

### PR DESCRIPTION
**Pull request by bprashanth**

### CODE REVIEW

File: test/e2e/testing-manifests/ingress/nginx/rc.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 0
       containers:
-      - image: gcr.io/google_containers/nginx-ingress-controller:0.8.3
+      - image: gcr.io/google_containers/nginx-ingress-controller:0.9.0-beta.1
```
Comment (ixdy) [inline test/e2e/testing-manifests/ingress/nginx/rc.yaml]: where is this defined? https://github.com/kubernetes/contrib/blob/master/ingress/controllers/nginx/Makefile#L4 is still 0.8.3.

Comment (aledbf) [inline test/e2e/testing-manifests/ingress/nginx/rc.yaml]: @ixdy here https://github.com/kubernetes/ingress/blob/master/examples/deployment/nginx/nginx-ingress-controller.yaml

Comment (ixdy) [inline test/e2e/testing-manifests/ingress/nginx/rc.yaml]: that's where it's used. I'm asking about the source for the image - it looks like the image was pushed without merging it into the repo, which makes it un-reproducible.

**Comment by ixdy** [comment]
oh, I'd missed that it moved out of `contrib/`. Are you going to send a PR to delete it from `contrib/`?

/approve

**Pull request merged**